src/reanimated2/hook/useAnimatedSensor.ts

```diff
@@ -52,18 +124,42 @@ export function useAnimatedSensor(
     isAvailable: false,
     config: {
       interval: 0,
+      adjustToInterfaceOrientation: false,
+      iosReferenceFrame: IOSReferenceFrame.XArbitraryCorrectedZVertical,
     },
+    interfaceOrientation: makeMutable<InterfaceOrientation>(0),
```

Comment:
Should we use `makeMutable` here or `makeShareable`?

Reply:
What's the difference between those two? Is it documented somewhere? I used it just because sensors used it already and orientation is very similar.

Reply:
Unfortunately, it's not documented as it's not in the public API. If you've copied it from a similar place, then that's okay.